cylc/flow/scheduler.py

```diff
@@ -903,11 +906,29 @@ def get_command_method(self, command_name: str) -> Callable:
         """Return a command processing method or raise AttributeError."""
         return getattr(self, f'command_{command_name}')
 
-    def queue_command(self, command: str, kwargs: dict) -> None:
-        self.command_queue.put((
-            command,
-            tuple(kwargs.values()), {}
-        ))
+    def queue_command(
+        self,
+        name: str,
+        args: list,
+        kwargs: dict,
+        log_lines: List[str],
+    ) -> str:
+        """Queue a command for action by the scheduler.
+
+        Return a unique command ID for provenance tracking.
+
+        """
+        uuid = str(uuid4())
+        LOG.info(f"{log_lines[0]} id={uuid}\n{log_lines[1]}")
```

Comment:
Not according to me!

Reply:
The two log lines get formatted in the resolver and passed through to this code.

In the resolver, it can get logged here:

https://github.com/cylc/cylc-flow/pull/5658/files/d2839b47149a61c3b88ea1acfb375335ad40d30e#diff-187d2487e55ce07803fa1a6597bbe848500119ef60daf535d01780a6be0bb5c0R719

According to this condition:

```python
if (
    method is not None
    and (
        command != "put_messages"
        or user != self.schd.owner
    )
)
```

Additionally, it will always get logged at the scheduler by this line.

Reply:
> According to this condition:

No, very next line is `return method(**kwargs)` - that block is for remaining internal methods in the resolver module, which don't go on to the scheduler command queue.